api/v2/helpers/bouquets/bouquets_helper.py

```diff
@@ -9,3 +9,12 @@ def query_all_bouquets():
         bouquets = BouquetsSchema(many=True).dump(all_bouquets)
 
     return bouquets
+
+
+# function to get a specific bouquet by id
+def get_bouquet(bouquet_id):
+    bouquet = BouquetsModel.query.filter_by(id=bouquet_id).first()
+    if bouquet:
+        return BouquetsSchema(many=False).dump(bouquet)
+
+    return False
```

Comment:
I think it would be better to return `None` than `False`. It's not good for a function to return two different types.